### DISCUSSION TITLE
Taxon Tiles: UI/UX enhancements

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#6007] [Feature Requests]
From [Discourse: ](https://discourse.specifysoftware.org/t/changing-taxon-tile-and-statistics-panel-behaviors/2167)
**Is your feature request related to a problem? Please describe.**

> Our preference for taxon tiles behavior is to have the tiles display the relative proportion of lots by the preferred name of their current determination only, rather than for the determined taxa themselves (see the above linked thread for more details on how tiles currently works, at least in our instance). Having this dichotomy be user-configurable (a switch for determined taxon or preferred taxon) settable at the user or collection level) would be ideal.
> Allowing collections to add fields to the query that is run when clicking on a tile would be great as well. Because the tiles don’t appear to be getting their count info from the associated queries, it shouldn’t impact load times for the graph itself.

> Allowing collections to add fields to the query that is run when clicking on a tile would be great as well. Because the tiles don’t appear to be getting their count info from the associated queries, it shouldn’t impact load times for the graph itself.

- Interacts with #303 

> If taxon tiles are not set to be the home page, there’s currently no sidebar menu option to view them. Having them added to preferences as a “visualization” tab, with the potential for other visualization options in the future, would be nice! (Of course, trees are also visualizations, but only of relationships, not of relative quantities.) It would also be interesting to allow similar “visualization+clickable query” UI with the contents of other tables, for example, agents, events, localities, geographies, storage, preptype, etc. Until then, simply having the option in preferences to get to the taxon tile page when it is not the homepage would be cool.

> Being able to zoom in on a section of the tiles, or change the way tiles are grouped, would make the visualization much more useful. An interface using a more hierarchical treemap that was divided or colored based on the ranks the user selected in a particular tree, for example.

**Describe the solution you'd like**

More preferences for taxon tiles: 
- choosing name that displays when hovering (preferred current vs determined name) 
- option to add taxon tiles to navigation menu

General taxon improvements: 
- scope to collection 
- allow to zoom in on parts of tiles

**Reported By**
Nate @ Ohio Mollusks

**Additional context**

> Overhauling these is fairly low on the list of features I’d want to see devs spend a ton of time on right now given more pressing issues (like batch edit and fixes to export encoding/aggregated field handling), but a few of the things below seem trivial to implement, and might really improve database usability and clarity for lay users.

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/changing-taxon-tile-and-statistics-panel-behaviors/2167/2
